src/administration/from_scratch.md

```diff
@@ -41,7 +41,10 @@ adduser lemmy --system --disabled-login --no-create-home --group
 
 Note:
 
-- Lemmy currently only supports non-SSL connections to databases. More info [here](https://github.com/LemmyNet/lemmy/issues/3007).
+- Lemmy currently supports TLS connections to databases with `sslmode=require`.
+  - By default `tls_enabled` will be false, so using SSL connections is entirely opt-in.
+  - However, when `tls_enabled` is enabled, the server identity will not be verified.
+  - More info [here](https://github.com/LemmyNet/lemmy/pull/3189).
```

Comment:
I dont like to link to pull requests from docs because the info will quickly get outdated. For example the initial PR text mentions that ssl connections use the os certificate store to verify connections, which is already wrong. So better to put any important info directly here.